traffic_ops/traffic_ops_golang/user/user.go

```diff
@@ -271,6 +271,8 @@ func (user *TOUser) Update(h http.Header) (error, error, int) {
 		return userErr, sysErr, errCode
 	}
 
+	user.RegistrationSent = &tc.TimeNoMod{Time: time.Now()}
```

Comment:
confused. updating user (PUT /users/:id) should not update registration_sent value.

Reply:
the ONLY way to change that value is by calling `POST users/register`

Reply:
oh wait, nm. i guess i'm looking at outdated code :)

Reply:
can u change the title of this PR? it confused me.

Reply:
this is old code.